src/client/Microsoft.Identity.Client/Instance/Discovery/RegionAndMtlsDiscoveryProvider.cs

```diff
@@ -73,7 +82,18 @@ private static string GetRegionalizedEnvironment(Uri authority, string region, R
             }
 
             requestContext.Logger.Info(() => $"[Region discovery] Regionalized Environment is : {region}.{host}. ");
-            return $"{region}.{host}";
+
+            // Decide whether to use MTLS or standard environment
```

Comment:
Unable to understand what is the difference between MTLS or standard env regional call

Reply:
when we started off it was providing a generic host for MTLS as well, but now we only use this as regional. But it is good to have the info in the logs IMHO. maybe I can word it differently